test/smoke/src/areas/preferences/preferences.test.ts

```diff
@@ -16,10 +16,10 @@ export function setup(logger: Logger) {
 			const app = this.app as Application;
 
 			await app.workbench.settingsEditor.openUserSettingsFile();
-			await app.code.waitForElements('.line-numbers', false, elements => !!elements.length);
+			await app.code.waitForElements('.margin .line-numbers', false, elements => !!elements.length);
 
 			await app.workbench.settingsEditor.addUserSetting('editor.lineNumbers', '"off"');
-			await app.code.waitForElements('.line-numbers', false, result => !result || result.length === 0);
+			await app.code.waitForElements('.margin .line-numbers', false, elements => !elements || elements.length === 0);
```

Comment:
Why did these ones need to be changed?

Reply:
I'll revert the selectors considering the old ones worked.